packages/create-sitecore-jss/src/common/prompts/styleguide.ts

```diff
@@ -1,11 +1,13 @@
+import { Answers, DistinctQuestion } from 'inquirer';
 import chalk from 'chalk';
 
-import { NextjsStyleguideAnswer } from './NextjsStyleguideAnswer';
-import { QuestionCollection } from 'inquirer';
+export interface StyleguideAnswer extends Answers {
+  language?: string;
```

Comment:
Can we call this LanguageAnswer instead? Or do you think there could be more added here in the future?

Reply:
Here can be added more, it's specifically for Styleguide-like samples